Soundex.h

```diff
@@ -3,6 +3,57 @@
 
 #include <string>
 
-std::string generateSoundex(const std::string& name);
+class Soundex {
```

Comment:
What is the use of Class when these is no state or why don't mark each  member function as static